ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh

```
@@ -62,9 +62,9 @@ set +o errexit
 RUN_EXIT_CODE=$?
 set -o errexit
 
-if [[ "${ENABLE_LOCAL_INDEX}" == "true" ]]; then
-    metrics_folder_name=$(find . -maxdepth 1 -type d -name 'collected-metric*' | head -n 1)
-    cp -r "${metrics_folder_name}" "${ARTIFACT_DIR}/"
+METRICS_FOLDER="collected-metrics-${UUID}"
+if [[ -d ${METRICS_FOLDER} ]]; then
+  cp -r ${METRICS_FOLDER} "${ARTIFACT_DIR}/"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)